### DISCUSSION TITLE
Table in DB selector

### DISCUSSION THREAD

**Discussion by Roy-Orbison** [adminerevo/adminerevo#38] [General]
Wondering about the reasoning behind [using tables for layout](https://github.com/adminerevo/adminerevo/blob/2be76a970ae461877e5504e6e6f7c784e3ec53ea/adminer/include/adminer.inc.php#L1043). It's also lost the `id="dbs"` attribute, and gained inline style rules. It seems to be a step backwards, given tables are more difficult to style and semantically for tabular data, so I don't understand the benefit.

**Comment by LionelLaffineur** [comment]
Hi @Roy-Orbison,
I have noticed that removing <p id="dbs"> could cause an issue to plugins or themes, so I have put it back for next release.
I agree with the styling without tables.  If we were in a bootstrap environment, I'd have done it directly.  In the way adminer layout is coded, to have the two selects aligned, I didn't find another way to do it than a table.
If you were so kind to propose something, I would appreciate very much and get inspired for further styling !
Regards,
L.

**Reply by Roy-Orbison** [reply]
Perhaps a table does suit, then. Other forms with aligned inputs, such as select filters, also use tables.
